lib/ddtrace/vendor/multipart-post/multipart.rb

```diff
@@ -0,0 +1,12 @@
+#--
+# Copyright (c) 2007-2013 Nick Sieger.
```

Comment:
We probably want the license in its entirety somewhere.

Including this full license at the top of every file is probably safest, otherwise maybe we can include `lib/ddtrace/vendor/LICENSE`? (not 100% sure which is best, but definitely think we need the full license somewhere)

```
Released under the MIT license.

Copyright, 2007-2013, by Nick Sieger.
Copyright, 2017, by Samuel G. D. Williams.
Copyright, 2019, by Patrick Davey.

Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Reply:
Yeah in that directory makes sense. I'll just add that... better safe than sorry.